src/components/DirectLinks/DirectLinks.js

```diff
@@ -0,0 +1,27 @@
+import React from 'react';
+import styles from './DirectLinks.module.scss';
+import { LINKS } from './DirectLinks.constants';
+import { ReactComponent as OutsideLink } from 'assets/icons/Outside-Link.svg';
```

Comment:
Maybe name this `ExternalLinkIcon` ? Outside isn't really a term used for external links.

Reply:
Modified it.